tests/integration/testdata/buildcmd/terraform/zip_based_lambda_functions_local_backend/py_build.sh

```diff
@@ -1,14 +1,22 @@
 src_code=$1
 build_path=$2
 output_name=$3
-echo "building ${src_code} into ${build_path}"
```

Comment:
Do these build scripts work on Windows too?

Reply:
I already discussed this point with @hawflau, and he will add a new project for the windows.